CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### Added
+
+- Add `identity_point` as method of composer [#572](https://github.com/dusk-network/plonk/pull/572)
```

Comment:
This is pull request #575. This references an issue with a similar name. Either change the number to 575, or `pull` -> `issue`.